sources/packages/web/src/views/aest/institution/ProfileCreate.vue

```diff
@@ -36,17 +36,22 @@ export default defineComponent({
   setup() {
     const router = useRouter();
     const snackBar = useSnackBar();
+    const { excludeExtraneousValues } = useFormioUtils();
     const processing = ref(false);
     const institutionProfileModel = ref({
       clientType: ClientIdType.AEST,
       mode: "create",
     } as FormModel);
 
-    const createInstitution = async (data: AESTCreateInstitutionAPIInDTO) => {
+    const createInstitution = async (data: unknown) => {
```

Comment:
why it was updated to `unknown`

Reply:
We can keep it mapped as `AESTCreateInstitutionAPIInDTO` or whatever we want. Either way, the form.io will not have the type enforced. The idea of having it as `unknown` was to ensure that this `data` would not be used anywhere before it is properly typed.
If need access to some property not mapped to `AESTCreateInstitutionAPIInDTO` we probably should have another type mapped.
If it creates more confusion as `unknown` I can revert it.